src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -203,6 +206,7 @@
     <Reference Include="System.Resources.ResourceManager" />
     <Reference Include="System.Memory" />
     <Reference Include="System.Runtime" />
+    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
```

Comment:
Is adding this dependency fine here for System.Net.Primitives?

Reply:
Same problem as with System.Private.Xml